<launch>

    <arg name="car_state_topic"           default="state" />
    <arg name="car_input_topic"           default="control_input" />
    <arg name="safe_car_input_topic"        default="safe_car_input_topic" />
    <arg name="safety_filter_config" default="$(find ros_safety_framework)/config/collision_avoidance.yaml" />

    <node pkg="ros_safety_framework" name="safety_filter_node" type="safety_filter_node" output="screen">
        <rosparam command="load" file="$(arg safety_filter_config)" />
        <remap from="~state" to="$(arg car_state_topic)" />
        <remap from="~input" to="$(arg car_input_topic)" />
        <remap from="~save_input" to="$(arg safe_car_input_topic)" />
    </node>

</launch>